pkg/api/resource/quantity.go

```diff
@@ -112,10 +112,19 @@ func MustParse(str string) Quantity {
 	return *q
 }
 
+// Scale is used for getting and setting the scaled value.
```

Comment:
comment could be more descriptive - is it a power of 10?  a power of an arbitrary base?